### PR TITLE
Update Alchemist configuration to include LS-9.

### PR DESCRIPTION
**Pull request by tebadi**
Upgrade Alchemist configuration to include Landsat-9.

### CODE REVIEW

File: prod/services/alchemist/ga_ls_fc_3/ga_ls8_ls9_fc_3.alchemist.yaml [hunk 1]
```diff
@@ -46,16 +47,45 @@ specification:
...
+          - 0.9499
+        pv:
+          - 2.77
+          - 0.9481
+        npv:
+          - -0.73
+          - 0.9578
 output:
-  location: s3://dea-public-data/derivative/
+  location: s3://dea-public-data-dev/derivative/
```
Comment (emmaai) [inline prod/services/alchemist/ga_ls_fc_3/ga_ls8_ls9_fc_3.alchemist.yaml]: output into dev?

File: prod/services/alchemist/ga_ls_fc_3/ga_ls8_ls9_fc_3.alchemist.yaml [hunk 2]
```diff
@@ -55,7 +85,7 @@ output:
     red: bs
     green: pv
     blue: npv
-  explorer_url: https://explorer.dea.ga.gov.au
+  explorer_url: https://explorer.dev.dea.ga.gov.au
```
Comment (lars-fillmore) [inline prod/services/alchemist/ga_ls_fc_3/ga_ls8_ls9_fc_3.alchemist.yaml:88]: This is a production config file, so it should reference the production explorer domain.

Comment (matt-fry-ga) [reply prod/services/alchemist/ga_ls_fc_3/ga_ls8_ls9_fc_3.alchemist.yaml:88]: Soz, that's my other profile.

Comment (tebadi) [reply prod/services/alchemist/ga_ls_fc_3/ga_ls8_ls9_fc_3.alchemist.yaml]: Good catch, fixed.

File: prod/services/alchemist/ga_ls_fc_3/ga_ls8_ls9_fc_3.alchemist.yaml (hunk 1 above)
Comment (tebadi) [reply prod/services/alchemist/ga_ls_fc_3/ga_ls8_ls9_fc_3.alchemist.yaml]: See the latest commit please.

**Review by JM-GA — APPROVED** [review]

**Pull request merged**